src/MeshGenController.cpp

```
@@ -0,0 +1,200 @@
+    emit busyChanged();
+}
+
+void MeshGenController::cancel()
+{
+    if (m_busy) {
+        m_cancel = true;
+        emit statusMessage(tr("Cancelling…"));
+    }
+}
+
+void MeshGenController::pickImageAndGenerate(int resolution, bool removeBackground)
```

Cppcheck (2.21.0)

[error] There is an unknown macro here somewhere. Configuration is required. If Q_ENUM is a macro then please configure it.

(unknownMacro)